packages/openapi3/src/cli/actions/convert/transforms/transform-component-schemas.ts

```diff
@@ -141,10 +141,20 @@ export function transformComponentSchemas(context: Context, models: TypeSpecData
     // Extract description and decorators from meaningful union members
     const unionMetadata = extractUnionMetadata(schema);
 
+    let decorators = [...getDecoratorsForSchema(schema), ...unionMetadata.decorators];
+
+    // Check if this is an SSE event schema - if so, replace @oneOf with @TypeSpec.Events.events
+    const schemaRef = `#/components/schemas/${name}`;
+    if (context.isSSEEventSchema(schemaRef)) {
+      // Remove @oneOf decorator if present and add @TypeSpec.Events.events
+      decorators = decorators.filter((d) => d.name !== "oneOf");
+      decorators.push({ name: "TypeSpec.Events.events", args: [] });
```

Comment:
just noticed this but we shouldn't include `TypeSpec.` namespace for those, it is implicitly working without.

Reply:
so just push `events` here (and update the tests accordingly) ?